package.json

```diff
@@ -200,20 +203,20 @@
     "stylelint-prettier": "^1.2.0",
     "stylelint-processor-styled-components": "^1.10.0",
     "stylis-plugin-rtl": "^1.0.0",
-    "terser-webpack-plugin": "^4.2.3",
+    "terser-webpack-plugin": "^5.1.1",
```

Comment:
Webpack v5 comes with the latest terser-webpack-plugin out of the box. So we might not even need to install it here.

Reply:
Still needed due to manual config